.github/workflows/rng.js

```diff
@@ -0,0 +1,44 @@
+const fs = require('fs');
+const crypto = require('crypto');
+const seedrandom = require('./seedrandom');
+
+function genSeedText() {
+    const { RNG_SEED } = process.env;
+    if (RNG_SEED) {
+        // Explicit seed is specified so use it
+        return RNG_SEED;
+    }
+    const { GITHUB_PR_NUMBER } = process.env;
+    if (GITHUB_PR_NUMBER) {
+        // GitHub pull request so use the GITHUB_REF as the seed to ensure consistency throughout PR lifecycle
+        return 'pr_' + GITHUB_PR_NUMBER;
+    }
+    // Otherwise generate a random seed. Note that we do not actually care
+    return 'seed_' + Date.now() + '_' + crypto.randomBytes(16).toString('hex');
+}
+
+function createRNG() {
+    const seedText = genSeedText();
+    const rng = new seedrandom(seedText);
+
+    console.log('::group::RNG Seed');
+    console.log('Initialized RNG with RNG_SEED = %s', seedText);
+    console.log('::endgroup::');
```

Comment:
What is the purpose of creating a group for a single entry?
It would be better to just have the value explicit in the logs so there's no need to make an extra click to see it.

Reply:
So that it's in the logs. It also shows up in the summary for direct copying: https://github.com/pgjdbc/pgjdbc/actions/runs/7410701405#summary-20163580422